src/common/components/signUpModal/signUpModal.component.spec.js

```diff
@@ -1221,28 +1223,6 @@ describe('signUpForm', function () {
       done()
       });
     });
-
-    it('should handle an error with $log and retry once', (done) => {
-      const error = new Error('Error signing in')
-      jest.spyOn($ctrl.$log, 'error').mockImplementation(() => {})
-      jest.spyOn($ctrl, 'reRenderWidget').mockResolvedValue()
-      const showSignInAndRedirect = jest.fn().mockRejectedValue(error)
-      const handleLoginRedirect = jest.fn()
-      $ctrl.oktaSignInWidget = {
-        showSignInAndRedirect,
-        authClient: {
-          handleLoginRedirect
-        }
-      }
-
-      $ctrl.signIn().then(() => {
-        expect(handleLoginRedirect).not.toHaveBeenCalled()
-        expect($ctrl.reRenderWidget).toHaveBeenCalledTimes(1)
-        expect($ctrl.$log.error).toHaveBeenCalledTimes(1)
-        expect($ctrl.$log.error).toHaveBeenCalledWith('Error showing Okta sign in widget.', error)
-        done()
-      })
-    })
```

Comment:
I also looked into this this morning. I'm unsure what has changed to prevent it from working. The signIn function caches the first response, which returns the token.

If we can get this text working again, it would be helpful, as we don't test whether a sign-in error is logged elsewhere.

Reply:
I tried that, but it seemed like the test was caching the first good response, even though I was telling it to error. Maybe you will have better luck than I did.

Reply:
I added it back in c7e5fa1dfe2cbaa2def56d98396613efc8136a1a